docs/sources/oncall-api-reference/teams.md

```diff
@@ -28,6 +28,7 @@ The above command returns JSON structured in the following way:
 ```json
 {
   "id": "TI73TDU19W48J",
+  "grafana_team_id": 123,
```

Comment:
Any reason not to call these `grafana_id` in both cases? (it seems clear enough and simpler to "automate" when using the API?)

Reply:
good point 👍 updated in https://github.com/grafana/oncall/pull/5075/commits/e5d1ae08405d21a7f95028e5a3512bff1dba93dd